versions.props

```diff
@@ -45,8 +45,7 @@ org.mockito:* = 5.12.0
 # Don't upgrade, we will remove this in a future release.
 # Users should depend on this plugin directly such that it will get updated by excavator.
 com.palantir.javaformat:gradle-palantir-java-format = 1.1.0
-# Newer spotless versions have issues resolving dependencies at configuration time
-com.diffplug.spotless:spotless-plugin-gradle = 6.6.0
```

Comment:
The issues that prompted this pin in https://github.com/palantir/gradle-baseline/pull/2312 is still open:

https://github.com/diffplug/spotless/issues/1380

Do we have reason to believe this issue is fixed?

Reply:
In testing this and coil mentioned in the issue. Coil still has it, this repo seems to currently not have it, and people seem to think its related to indirect dependencies being loaded. Gradle does correct the issue if it comes up, it just causes a perf hit. Without bumping spotless, repos using OpenJFx and some other libs can compile at all. If we do not update this lib, repos with that issue will be forced to disable baseline-spotless entirely.

Reply:
The bug reports say that gradle has a catch for this and the worst case scenario is: if there are indirect dependencies and it has to work through them, hitting this regression, then it is a small perf hit, but doesnt fail builds. Where as without this we cant compile UI apps at all. :/

Reply:
Is there a path to get spotless updated here?